FrontEnd/src/components/ProfilePage/ProfilePageComponents/ProfileContent.js

```diff
@@ -10,6 +10,22 @@ import StartupInfo from '../FormComponents/StartupInfo';
 import UserInfo from '../FormComponents/UserInfo';
 import ProfileFormButton from '../UI/ProfileFormButton/ProfileFormButton';
 
+if(sessionStorage.getItem("isAuthenticated")){
+const token  = "Token " + sessionStorage.getItem('auth_token');
+axios
+  .get('http://localhost:8000/api/getuserid/',
+  {withCredentials: true,
+  headers: {
+        'Authorization': token}})
+  .then(function (response) {
+    console.log(response.data);
+    console.log(response.status);
+  })
+  .catch((error) => {
+    console.log(error);
+  });
+}
+
```

Comment:
I would highly recommend using redux-thunk for managing asynchronous requests. You may utilize it in pair with [redux-toolkit](https://redux-toolkit.js.org/introduction/getting-started) to simplify work lift. If the team doesn't have enough time to learn redux I would recommend using [react-query](https://tanstack.com/query/latest/docs/react/overview) for managing API requests which is easier to start

Reply:
Thank you for feedback. My task was to connect using axios, to write little examples of requests for teammates. I'll tell about your advice to use react-query to my teammates.